k8s/stubbornjava.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stubbornjava-deployment
+  labels:
+    app: sj-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sj-web
+  template:
+    metadata:
+      labels:
+        app: sj-web
+    spec:
+      containers:
+        - name: sj-web
+          image: containers.pkg.github.com/stubbornjava/stubbornjava-webapp:prod
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+          env:
+            - name: ENV
+              value: "prod"
+          volumeMounts:
+            - name: config-volume
+              mountPath: /app/config/
+      volumes:
+        - name: config-volume
+          configMap:
+            name: sj-web-config-prod
+            items:
+            - key: sjweb.production.conf
+              path: sjweb.production.conf
+      imagePullSecrets:
+        - name: ghregistry
+
```

Comment:
Before long you'll want to define a `resources` block. By default your QoS will be `BestEffort` and you'll likely experience CPU throttling and/or eviction. At first it can seem confusing why this happens, but there's a good reason for it and at the end of the day being up front with the scheduler about resources is the way to go.